examples/features/csm_observability/README.md

```diff
@@ -0,0 +1,30 @@
+# CSM Observability
+
+This examples shows how to configure CSM Observability for a binary, and shows
+what type of telemetry data it can produce for certain RPC's with additional CSM
+Labels. The Client accepts configuration from an xDS Control plane as the
+default address that it connects to is "xds:///helloworld:50051", but this can
+be overridden with the command line flag --addr.
+
+## Try it (locally if overwritten xDS Address)
+
+```
+go run server/main.go
+```
+
+```
+go run client/main.go
+```
+
+```
+curl localhost:9464/metrics
+```
+
+# Building
```

Comment:
I don't get a clear idea if this `Building` step should happen before the `Try it` or after, or does not matter?

Reply:
Does not matter. It's weird because this example doesn't work locally by default, but it is still an example. So I just listed the steps in our other examples and mentioned you have to overwrite xds target string. The building step is from C++, and perhaps it's there so users can build the image and deploy it themselves?

Reply:
If the example is not supposed to work by default, then we should state so. And also state how users can make it work, if they want to.

Reply:
Done, added "Note that this example will not work by default, as the client uses an xDS
Scheme and thus needs xDS Resources to connect to the server. Deploy the built
client and server containers within Cloud Service Mesh in order for this example
to work, or overwrite target to point to :<server serving port>."